settings.py

```diff
@@ -1557,6 +1557,11 @@
             'class': 'pillowtop.pillow.interface.ConstructedPillow',
             'instance': 'corehq.pillows.case_search.get_case_search_to_elasticsearch_pillow',
         },
+        # {
+        #     'name': 'LedgerToElasticsearchPillow',
+        #     'class': 'pillowtop.pillow.interface.ConstructedPillow',
+        #     'instance': 'corehq.pillows.ledger.get_ledger_to_elasticsearch_pillow',
+        # },
```

Comment:
yes

On 12 May 2016 at 14:16, Cory Zue <notifications@github.com> wrote:

> In settings.py
> <https://github.com/dimagi/commcare-hq/pull/11580#discussion_r63009489>:
>
> > @@ -1557,6 +1557,11 @@
> >              'class': 'pillowtop.pillow.interface.ConstructedPillow',
> >              'instance': 'corehq.pillows.case_search.get_case_search_to_elasticsearch_pillow',
> >          },
> > +        # {
> > +        #     'name': 'LedgerToElasticsearchPillow',
> > +        #     'class': 'pillowtop.pillow.interface.ConstructedPillow',
> > +        #     'instance': 'corehq.pillows.ledger.get_ledger_to_elasticsearch_pillow',
> > +        # },
>
> confirming you want this merged without enabling the pillow yet?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11580/files/18e132334e2857d8de0017e64bca16e08ffce271#r63009489>
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa